src/components/map/Popups/StudyPopup.tsx

```diff
@@ -1,9 +1,22 @@
 import React from "react";
+import { Button } from 'semantic-ui-react'
 import { AirtableRecord } from "types";
 import Translate from 'utils/translate/translateService';
 import { getGeography, getPossibleNullDateString } from 'utils/utils';
+import InformationIcon from "../../shared/InformationIcon";
 
 export default function StudyPopup(record: AirtableRecord) {
+    const getVaccinationIconText = (vaccinationPolicy: number) => {
+        const vaccinationPolicyMapping: Record<number, string> = {0: "No availability",
```

Comment:
Need to incorporate this into our translation system

Reply:
I'm pretty sure we actually aren't using this function at all anymore so im going to remove